app/steps/ui/screeners/willleft/index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const EligibilityValidationStep = require('app/core/steps/EligibilityValidationStep');
+const content = require('app/resources/en/translation/screeners/willleft');
+const pageUrl = '/will-left';
+const fieldKey = 'left';
+const FeatureToggle = require('app/utils/FeatureToggle');
+
+class WillLeft extends EligibilityValidationStep {
+
+    static getUrl() {
+        return pageUrl;
+    }
+
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
+        super.handlePost(ctx, errors, formdata, session, hostname, featureToggles);
+        ctx.isToggleEnabled = FeatureToggle.isEnabled(featureToggles, 'intestacy_screening_questions');
+        return [ctx, errors];
+    }
+
+    getContextData(req, res) {
+        const featureToggles = {
+            isToggleEnabled: 'intestacy_screening_questions'
+        };
+        return super.getContextData(req, res, pageUrl, fieldKey, featureToggles);
```

Comment:
Because when submitting an answer, `getContextData` gets executed before `handlePost` and is therefore still unaware of the status of any feature toggle, which means it matches the wrong option in `nextStepOptions`.
Makes sense?

Reply:
If I comment out `getContextData()` in this class then the redirection to the next/stop page still works fine with the toggle on and off so in what scenario does this code get called?